src/components/Layouts/Navbar.js

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] I request that you re-enable the linters and fix them instead. They help to create good quality code that is also well formatted.